packages/forms-web-app/src/controllers/guidance-pages.js

```diff
@@ -0,0 +1,13 @@
+const { VIEW } = require('../lib/views');
+
+exports.getBeforeAppeal = (_, res) => {
+  res.render(VIEW.GUIDANCE_PAGES.BEFORE_APPEAL, {
+    currentUrl: '/before-you-appeal',
+    heading: 'Appeal a householder planning decision',
+    nextPage: {
+      text: 'When you can appeal',
+      url: '/when-you-can-appeal',
+    },
+    title: 'Before you appeal - Appeal a householder planning decision - GOV.UK',
```

Comment:
are these things being set this way due to us also needing this page in the questionnaire project / somewhere else?

Reply:
The guidance pages are used 4 or 5 times in the project, so made it this way to make it more easily reusable